mcp_servers/tasks.py

```
@@ -0,0 +1,421 @@
+        }
+
+    def _tool_delete(self, args: Dict[str, Any]) -> Dict[str, Any]:
+        """Delete a task."""
+        deleted = self.task_manager.delete_task(args["task_id"])
+
+        return {
+            "success": deleted,
+            "message": "Task deleted" if deleted else "Task not found"
+        }
+
+    def _tool_stats(self, args: Dict[str, Any]) -> Dict[str, Any]:
```

Ruff (0.14.14)

[warning] Unused method argument: `args`

(ARG002)